python/ray/worker.py

```diff
@@ -710,10 +720,11 @@ def init(
             and the API is subject to change.
 
     Returns:
-        If the provided address included a protocol (e.g. "ray://1.2.3.4")
-        then a ClientContext is returned with information such as settings,
-        server versions for ray and python, and the dashboard_url.
-        Otherwise, returns address information about the started processes.
+        If the provided address included a protocol, for example
+        "ray://1.2.3.4:10001", then a ClientContext is returned with
```

Comment:
Maybe something like the following to make it clear what the protocol portion is.
```
"ray://" in "ray://1.2.3.4:10001"
```